Tasks/UniversalPackagesV0/universaldownload.ts

```diff
@@ -0,0 +1,128 @@
+import * as tl from "vsts-task-lib";
+import {IExecSyncResult, IExecOptions} from "vsts-task-lib/toolrunner";
+
+import * as telemetry from "utility-common/telemetry";
+import * as artifactToolRunner from "./Common/ArtifactToolRunner";
+import * as artifactToolUtilities from "./Common/ArtifactToolUtilities";
+import * as auth from "./Common/Authentication";
+
+export async function run(artifactToolPath: string): Promise<void> {
+    let buildIdentityDisplayName: string = null;
+    let buildIdentityAccount: string = null;
+    try {
+        // Get directory to publish
+        let downloadDir: string = tl.getInput("downloadDirectory");
+        if (downloadDir.length < 1)
+        {
+            //todo
+            tl.warning(tl.loc("Info_NoPackagesMatchedTheSearchPattern"));
```

Comment:
???

Reply:
Oh, the resource string is just poorly named, the message "No directory matched the search pattern." is somewhat reasonable but could be better. there's a todo there though?

Reply:
yeah, I just missed my todo. Sorry about that and thanks for pointing this.